federation/rest.yaml

```diff
@@ -5,79 +5,107 @@
 # Next step: swagger.
 
 endpoints:
-  - title: Node identity
-    path: "/node/identity"
-    entrypoint: identity
-    authentication: []
+  - title: Federation node handshake
+    path: "/nodes/{nodeIDA}/handshake"
+    entrypoint: nodeHandshake
+    parameters:
+      path:
+      - name: nodeIDA
```

Comment:
why bothering with A/B here?

Reply:
Made sense when I was writing this, but now it doesn't anymore...